ginga/misc/plugins/Cuts.py

```diff
@@ -399,8 +483,22 @@ def redo(self):
         """This is called when a new image arrives or the data in the
         existing image changes.
         """
+        # Get reference to MultiDim
+        chname = self.fv.get_channelName(self.fitsimage)
+        chinfo = self.fv.get_channelInfo(chname)
+        self.mdim_ref = chinfo.opmon.getPlugin('MultiDim')
+
```

Comment:
I don't think `mdim_ref` is getting set if you open `Cuts` *after* loading an image.  Then it raises an exception (I've marked it lower down) because it is not defined.

In general, I think the approach of trying to check what another plugin is doing and work around that is probably not the correct approach.